src/common/bots/Bot.cpp

```diff
@@ -0,0 +1,73 @@
+#include "Bot.hpp"
+
+Bot::Bot(){};
+
+/* This strategy will move towards the closest mine which has a resource the bot needs to win.
```

Comment:
docstring comment style is 
```
/**
 * stuff
 */
```

:P

Reply:
this is what clang format does though :0